applications/nrf5340_audio/src/modules/Kconfig

```diff
@@ -116,4 +120,9 @@ config BUTTON_PUBLISH_STACK_SIZE
 	default 300
 
 endmenu # Stack sizes
+menu "SD card playback"
+config SD_CARD_PLAYBACK
+	bool "If true, lc3 playback is enabled"
+	default n
+endmenu # Lc3 playback
```

Comment:
We might replace "lc3" to "LC3 encoded file"

Reply:
I changed it to "sd card playback". I just forgot to change the name after I switched the module name from "LC3 playback"

Reply:
Since it can also play .wav files